packages/edit-widgets/src/components/sidebar/style.scss

```diff
@@ -21,59 +21,74 @@
 	}
 }
 
+// This tab style CSS is duplicated verbatim in
+// /packages/components/src/tab-panel/style.scss
 .components-button.edit-widgets-sidebar__panel-tab {
```

Comment:
Asking for future reference — Is there an easy way to test this widget screen? Do I need some kind of widget-enabled theme?

Reply:
Any default theme before 2022 should have widgets enabled.

Reply:
Ah, got it. Thank you!